planproexporter/planproxml/edgexml.py

```diff
@@ -12,7 +12,7 @@ def get_top_edge_xml(edge: Edge):
         connection_b = "Ende"
         if len(edge.node_b.connected_nodes) > 1:
             connection_b = edge.node_b.get_anschluss_of_other(edge.node_a).name
-        return f"            <TOP_Kante> <!-- {edge.node_a.uuid} to {edge.node_b.uuid} -->\n" \
+        return f"            <TOP_Kante> <!-- {edge.node_a} to {edge.node_b} -->\n" \
```

Comment:
Before introducing this, we need to agree on a method of ensuring that the name is valid in terms of planpro as there are restrictions in place for both what is considered a valid Wert for Identitaet and Name. We are currently relying on setting the name to the OSM node id, which violates this constraint. I think we should introduce a separate method that returns a valid Wert.

Reply:
Since this is only a comment for readability, is this really necessary here? So this comment will never be validated by PlanPro. In case of the CLI importer, these are usually `n1`, `n2`, ..., which is way better readable than a UUID.

Reply:
oh, I misread that when reviewing the change the last time. Sure, the comment is fine